_spinnaker/terraform_integration.md

```diff
@@ -4,114 +4,113 @@ title: Terraform Integration
+If you decide to enable this feauture and have feedback you'd like to submit, please let us know at [go.armory.io/ideas](go.armory.io/ideas)! We're constantly iterating on customer feedback to ensure that the features we build make your life easier!
 
 ## Prerequisites
 
-1. An Armory Spinnaker installation (version 2.3.x or above) running on Kubernetes and installed via Armory Halyard. ([instructions](/spinnaker/install)). If you haven't updated Armory Halyard in a while, you'll need to do so to get access to these new features.
-1. Credentials (in the form of basic auth) to your Terraform git repository.  This can take one of several forms:
-    1. If your Terraform repo is in Github, you can use a Personal Acccess Token (potentially associated with a service account) as the 'token'.  See [this documentation](https://blog.github.com/2013-05-16-personal-api-tokens/) for details on how to generate this token in Github.
-    1. If your Terraform repo is in BitBucket, you can use a username/password that has access to your BitBucket repo
-1. If you want to use Terraform Input Variable File `tfvar` capability, a separate enabled artifact provider (such as the Github, BitBucket, or HTTP artifact provider) that is able to pull your `tfvar` file(s).
+* Credentials (in the form of basic auth) to your Terraform Git repository.  This can take one of several forms:
+    * If your Terraform repo is in GitHub, use a Personal Acccess Token (potentially associated with a service account) as the 'token'.  Generate this token in your GitHub settings.
+    * If your Terraform repo is in BitBucket, use a username/password that has access to your BitBucket repo.
+* To use Terraform Input Variable Files (`tfvar`), you must have a separate artifact provider (such as the GitHub, BitBucket, or HTTP artifact provider) that can pull your `tfvar` file(s). Additionally, the credentials for must be configured in both places: the Terraform Integration and the artifact provider.
 
 ## Overview
 
-The Terraformer integration interacts with your source repository as follows:
-
-1. It will use the generic `git/repo` using basic auth creds provided directly to Terraformer (this can be a Github token or a BitBucket username/password combination) to pull a full directory from your Git repository
-1. Optionally, it will use a traditional Spinnaker artifact provider (Github, BitBucket, HTTP) to pull in a `tfvars`-formatted variable file.  This document covers Github and BitBucket.
-1. These credentials must be configured in both places (directly with Terraformer, and also with the artifact provider)
+Armory Spinnaker's Terraform Integration interacts with a source repository you specify to deploy infrastructure.
 
-## Installation
+At a high level, the Terraform Integration performs the following actions during a Terraform Integration stage:
```

Comment:
Using `Terraform Integration` twice feels repetitive. How about:
`At a high level, the Terraform integration performs the following actions during execution:`